apps/client/src/features/rundown/RundownEntry.tsx

```diff
@@ -75,33 +49,6 @@ export default function RundownEntry(props: RundownEntryProps) {
 
   const actionHandler = useMemoisedFn((action: EventItemActions, payload?: number | FieldValue) => {
     switch (action) {
-      case 'event': {
```

Comment:
question: what was the motivation for removing these actions?

Reply:
They are not used and allow us to remove quite a few props from the component